apps/spruce/src/pages/image/GeneralTable/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useMemo, useRef } from "react";
+import { useQuery } from "@apollo/client";
+import { useLeafyGreenTable, LGColumnDef } from "@leafygreen-ui/table";
+import { StyledRouterLink, WordBreak } from "components/styles";
+import { BaseTable } from "components/Table/BaseTable";
+import { getTaskRoute } from "constants/routes";
+import { useToastContext } from "context/toast";
+import {
+  ImageGeneralQuery,
+  ImageGeneralQueryVariables,
+} from "gql/generated/types";
+import { IMAGE_GENERAL } from "gql/queries";
+import { getDateCopy } from "utils/string";
+
+type GeneralInfo = {
+  property: string;
+  value: React.ReactNode | string;
+};
+
+type GeneralTableProps = {
+  imageId: string;
+};
+
+export const GeneralTable: React.FC<GeneralTableProps> = ({ imageId }) => {
+  const dispatchToast = useToastContext();
+  const { data: imageData, loading } = useQuery<
+    ImageGeneralQuery,
+    ImageGeneralQueryVariables
+  >(IMAGE_GENERAL, {
+    variables: { imageId },
+    onError(err) {
+      dispatchToast.error(
+        `There was an error loading the image general table: ${err.message}`,
+      );
+    },
+  });
+
+  const image = useMemo(() => imageData?.image, [imageData?.image]);
+
+  const tableData = useMemo(
```

Comment:
nit. Could we just call this `data`?
```suggestion
  const data = useMemo(
```